applications: nrf5340_audio: Added config for printing BASE (#28205)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

nRF5340 Audio
-------------

> * Added config option ``CONFIG_BT_AUDIO_BROADCAST_BASE_PRINT`` to allow printing the contents of the BASE when received.

Reviewer: ```suggestion
* Added the :kconfig:option:`CONFIG_BT_AUDIO_BROADCAST_BASE_PRINT` kconfig option to print the contents of the BASE when it is received.
```

Reviewer: @alexsven Could you please verify if this is an actual Kconfig option or just an application option? I'm getting conflicting info on this one

Author: Not sure I understand the question. It is a config that is defined here: https://github.com/nrfconnect/sdk-nrf/pull/28205/changes#diff-b592afcff78eaab32a130e3dee9f6765909a63d527402407c3e151e06d7ab42dR268 in a Kconfig file, but in the application. So if that makes it a Kconfig option or an application option I'm not sure I see the difference

>   This is useful for debugging purposes. Only valid for the :ref:`broadcast sink application <nrf53_audio_broadcast_sink_app>`.

Reviewer: ```suggestion
  This option is useful for debugging purposes.
  It is only valid for the :ref:`broadcast sink application <nrf53_audio_broadcast_sink_app>`.
```
